engine/README.md

```diff
@@ -198,3 +198,66 @@ mono bee.exe mac
 ```
 
 
+## build android sdk on mac
```

Comment:
try to make the section names and structures similar to those of other platforms